.gitignore

```diff
@@ -7,3 +7,5 @@ venv/
 .venv/
 .pytest_cache/
 **__pycache__/
+app/db.sqlite3
```

Comment:
Consider whether you need to exclude both `app/db.sqlite3` and `db.sqlite3`. If your project only uses one of these paths, you can remove the unnecessary entry to keep the `.gitignore` file clean.